### PR TITLE
Refactors ad rendering in articles

### PR DESCRIPTION
**Pull request by l2succes**
Fixes:
- https://artsyproduct.atlassian.net/browse/GROW-1258
- https://artsyproduct.atlassian.net/browse/GROW-1248

This switches our ad rendering library for HTLAds, Hashtag Labs' wrapper around Google GPT to [`react-gpt`](https://github.com/nfl/react-gpt). Because HTLAds was not compatible with our React SSR rendering requirements.

Namely, the ads would render and then disappear because React was not aware of the rendered ads. We implemented a hack that only worked partially. So a decision was made to move away from HTLAds in favor of a solution optimized for the React rendering lifecycle.

We also update the UI to match the specs given.

TODO:
- [x] fix tests
- [x] fix spacing

### CODE REVIEW

File: src/Components/Publishing/Display/NewDisplayCanvas.tsx [hunk 1]
```diff
@@ -17,29 +17,23 @@ export const NewDisplayCanvas: SFC<DisplayCanvasProps> = props => {
...
-        <div
-          className="htl-ad"
-          data-unit={adUnit}
-          data-sizes={adDimension}
-          data-eager
-        />
-      </DisplayCanvasContainer>
-    </ErrorBoundary>
+    <DisplayCanvasContainer flexDirection="column" m="auto" p={2}>
+      <GPT adUnitPath={`/21805539690/${adUnit}`} slotSize={[width, height]} />
```
Comment (l2succes) [inline src/Components/Publishing/Display/NewDisplayCanvas.tsx]: TODO: We should move our ad account ID to a env variable

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 732b635944567cc6c2c3637f113c8e066ac474af

https://deploy-preview-2440--artsy-reaction.netlify.com

File: src/Components/Publishing/Display/NewDisplayCanvas.tsx [hunk 2]
```diff
@@ -17,29 +17,23 @@ export const NewDisplayCanvas: SFC<DisplayCanvasProps> = props => {
...
+    </DisplayCanvasContainer>
   )
 }
 
 const DisplayCanvasContainer = styled(Flex)`
   min-height: fit-content;
-  max-width: 1250px;
   box-sizing: border-box;
+  background: ${color("black5")};
+  text-align: center;
```
Comment (damassi) [inline src/Components/Publishing/Display/NewDisplayCanvas.tsx]: `justifyContent='center'`

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 1afe902263098f3684babe154403c8d600c7a812

https://deploy-preview-2440--artsy-reaction.netlify.com

**Comment by damassi** [comment]
Much nicer than a bunch of custom code 👍

**Comment by l2succes** [comment]
The UI has been updated to match the spec and we also consolidate `NewDisplayCanvas` and `NewDisplayPanel` into a shared component `DisplayAd` as the logic is now identical. 

DisplayCanvas
Screen Shot 2019-05-21 at 12 26 26 PM

DisplayPanel
Screen Shot 2019-05-21 at 12 26 35 PM

**Comment by artsyit** [comment]
:rocket: PR was released in v16.12.4 :rocket:

**Pull request merged**